### PR TITLE
StringC: Append null terminator even if the string is empty

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**

### CODE REVIEW

**Review by mihails-strasuns-sociomantic — COMMENTED** [review]
Looks like a bug fix to me tbh (== v2.8.x base material).

But of course there always can be someone relying on the fact that for some cases `toCString` does not return a c string.. ;)

**Comment by mathias-lang-sociomantic** [comment]
Good point, changed base

File: relnotes/stringc-nullterm.migration.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* `ocean.text.util.StringC : StringC.toCString`
+
+  This function will now properly append a null terminator, even for empty string.
```
Comment (mihails-strasuns-sociomantic) [inline relnotes/stringc-nullterm.migration.md]: Don't need release notes now that it is based on point branch

**Comment by mathias-lang-sociomantic** [comment]
Removed relnotes

**Review by mihails-strasuns-sociomantic — APPROVED** [review]

**Pull request merged**